src/components/common/modal/modals.tsx

```diff
@@ -0,0 +1,36 @@
+import { EModalType, TModalProps } from "src/context-providers";
+import { useModal } from "src/context-providers/display/modal-hook";
+import { EmptyModal } from "./empty-modal";
+import { ScrollableModal } from "./empty-modal.styles";
+
+export const Modals = () => {
+    // =========================================================================
+    // CONST, STATE, REFS
+    // =========================================================================
+    const { modals, hideModal } = useModal();
```

Comment:
remove unused function

```suggestion
    const { modals } = useModal();
```